packages/components/src/index.ts

```diff
@@ -45,6 +45,7 @@ export { default as CountCard } from './highlight-cards/count-card';
 export { default as AnnualHighlightCards } from './highlight-cards/annual-highlight-cards';
 export { default as NoticeBanner } from './notice-banner';
 export { default as MobileHighlightCardListing } from './highlight-cards/mobile-highlight-cards';
+export { default as TooltipContent } from './highlight-cards/tooltip-content';
```

Comment:
Recognizing that this is likely temporary, I wonder if we should add a qualifier to clarify that this is tooltips specific to highlight cards? Or not expose it at all?

Reply:
OK, here's the last bit that moves them out: #103799